phpunit.xml

```diff
@@ -13,4 +13,9 @@
             <directory>tests</directory>
         </testsuite>
     </testsuites>
+    <filter>
+        <whitelist>
+            <directory suffix=".php">./src</directory>
```

Comment:
It is for running phpunit code coverage otherwise you will get error
```
Error:         Incorrect whitelist config, no code coverage will be generated.
```